deployment/terraform/network.tf

```diff
@@ -0,0 +1,17 @@
+module "vpc" {
+  source = "github.com/azavea/terraform-aws-vpc?ref=2.0.1"
+
+  name                       = "StormdrainMarking-${var.environment}"
```

Comment:
PWD reference here, but also this naming convention was imposed on us by PWD. For Raster Foundry we can change it up to something like `vpcRasterFoundry${var.environment}`.